apps/nextjs/src/app/[locale]/manage/apps/edit/[id]/_app-edit-form.tsx

```diff
@@ -55,6 +54,7 @@ export const AppEditForm = ({ app }: AppEditFormProps) => {
       buttonLabels={{
         submit: t("common.action.save"),
       }}
+      showBackToOverview={false}
```

Comment:
Why do we hide this button here?

Reply:
Because it will redirect to ``/manage/apps`` which is not what we want in the quick add

Reply:
But the _app-edit-form is only used on the `apps/edit/{id}` page where it would make sense to show that button, no?